internal/adapters/email/smtp_sender.go

```diff
@@ -11,44 +13,51 @@ import (
 	"gopkg.in/gomail.v2"
 )
 
+// SMTPAdapter schema for email smtp adapter.
 type SMTPAdapter struct {
 	config        *config.Config
 	templateCache map[string]*template.Template
 	templateMutex sync.RWMutex
 }
 
+// NewSMTPAdapter act as smtp sender constructor.
 func NewSMTPAdapter(cfg *config.Config) ports.EmailSender {
 	return &SMTPAdapter{
 		config:        cfg,
 		templateCache: make(map[string]*template.Template),
 	}
 }
 
-func (a *SMTPAdapter) SendOTPEmail(to, otp string) error {
+// SendOTPEmail uses markdown template and otp code to send an email.
+func (a *SMTPAdapter) SendOTPEmail(sendTo, otp string) error {
```

Comment:
The function name should change to `sendEmail` 
The variable changes to `recepient`